MapleServer2/Packets/FieldPacket.cs

```diff
@@ -236,6 +237,23 @@ public static Packet AddNpc(IFieldObject<Npc> npc)
             return pWriter;
         }
 
+        internal static Packet AddInteractActors(ICollection<IFieldObject<InteractActor>> actors)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.INTERACT_OBJECT);
+
+            pWriter.WriteByte(0x08);  // There are different types here.
+            pWriter.WriteInt(actors.Count);
+            foreach (IFieldObject<InteractActor> actor in actors)
+            {
+                pWriter.WriteShort((short) actor.Value.Uuid.Length);
+                pWriter.WriteString(actor.Value.Uuid);
+                pWriter.WriteByte(1);
+                pWriter.WriteByte(1);
+            }
+
+            return pWriter;
+        }
```

Comment:
Maybe move this packet to another file dedicated only for INTERACT_OBJECT packets

Reply:
Done in latest PR commit.